src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,38 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        System.out.println("Create five drivers.");
+        Driver john = new Driver("John", "AAA-00001");
+        Driver kate = new Driver("Kate", "AAA-00002");
+        Driver pamela = new Driver("Pamela", "AAA-00003");
+        Driver steve = new Driver("Steve", "AAA-00004");
+        Driver adam = new Driver("Adam", "AAA-00005");
+        driverService.create(john);
+        driverService.create(kate);
+        driverService.create(pamela);
+        driverService.create(steve);
+        driverService.create(adam);
+        driverService.getAll().forEach(System.out::println);
+
+        System.out.println("Get driver with id: 1.");
+        System.out.println(driverService.get(1L));
```

Comment:
```suggestion
        System.out.println(driverService.get(john.getId()));
```

I think It's better not to use "magic numbers" in code 🙂